front/src/applications/editor/components/EntitySumUp.tsx

```diff
@@ -132,14 +132,8 @@ function getSumUpContent(
       }
       break;
     }
-    // @ts-expect-error: Here we only deal with the installation_type, the rest is handled with BufferStop and Detector.
-    case 'Signal': {
-      const signal = entity as SignalEntity;
-      if (signal.properties.extensions.sncf.installation_type) {
-        type += ` - ${signal.properties.extensions.sncf.installation_type}`;
-      }
-    }
     // eslint-disable-next-line no-fallthrough
+    case 'Signal':
     case 'BufferStop':
     case 'Detector': {
       // This cast is OK since both buffer stops and detectors have same
```

Comment:
Could you merge this comment with the next line ? :)

Reply:
Hm, what do you mean?

Reply:
Resolved this in-person: the comment fits on a single line. While at it, also adjusted the comment to make it clear that signals also have the same props.